### ISSUE TITLE
Does "privacy.clearOnShutdown.siteSettings = TRUE" clear also local storage?

### ISSUE THREAD

**Issue by crssi** [ghacksuserjs/ghacks-user.js#119]
Just a quicky, since I am curios and if anyone happens to know, before digging into.

Does "privacy.clearOnShutdown.siteSettings = TRUE" clear also local storage?

Thank you
Cheers

**Comment by Thorin-Oakenpants** [comment]
I don;t think so. It clears site permissions and things like zoom levels. Storage is a different "container". I could possibly test it, but I have virtually nothing with any local storage, if anything. I don't mind copying my current portable FF and wiping my site settings to test - but I need some local storage - where can I get some?

**Comment by Thorin-Oakenpants** [comment]
Hah .. https://www.wolframalpha.com/ creates dom storage elements

**Comment by earthlng** [comment]
Idk if you know JS but if you do here's the [code](https://dxr.mozilla.org/mozilla-central/source/browser/base/content/sanitize.js#445) for siteSettings sanitize

**Comment by fmarier** [comment]
I think local storage is cleared with cookies, but I haven't checked recently.

**Comment by Thorin-Oakenpants** [comment]
That makes sense francois. I know Cookie Controller clears and handles dom storage and cookies together. It also has context menu items to clear dom storage / cookies separately (indeed, cookies can be handled via the FF ui), so they're not intrinsically (or tragically?) tied to each other.

**Comment by crssi** [comment]
Just came back. Thank you all
